core/src/main/java/com/graphhopper/routing/profiles/BitEncodedValue.java

```diff
@@ -0,0 +1,54 @@
+package com.graphhopper.routing.profiles;
+
+/**
+ * This class provides easy access to just one bit.
+ */
+public final class BitEncodedValue extends IntEncodedValue {
```

Comment:
A boolean usually uses 1byte (roughly) in the JVM, so I wanted to differentiate here as we really use just 1 bit, but not that important and we can change to Bool or Boolean